src/applications/vaos/components/FacilityPhone.jsx

```diff
@@ -12,8 +13,7 @@ export default function FacilityPhone({
   if (!contact) {
     return null;
   }
-
-  const [number, extension] = contact.split('x');
```

Comment:
Removing this constant array because main facility phone number do not have an extension.